dist/index.html

```diff
@@ -13,52 +13,31 @@ <h2>Leaderboard</h2>
       <div class="row">
         <section class="scores column">
           <h2>Recent scores</h2>
-          <button type="button">Refresh</button>
-          <table>
-            <tr>
-              <td>Name: 100</td>
-            </tr>
-            <tr class="scores-tr">
-              <td>Name: 20</td>
-            </tr>
-            <tr>
-              <td>Name: 50</td>
-            </tr>
-            <tr class="scores-tr">
-              <td>Name: 78</td>
-            </tr>
-            <tr>
-              <td>Name: 125</td>
-            </tr>
-            <tr class="scores-tr">
-              <td>Name: 77</td>
-            </tr>
-            <tr>
-              <td>Name: 42</td>
-            </tr>
-          </table>
+          <button id="refresh" type="button">Refresh</button>
+          <table id="games"></table>
         </section>
-        <section class="column">
+        <section class="add column">
           <h2>Add your score</h2>
           <form>
-            <div>
+            <span id="message"></span>
+            <div class="pad-btm">
               <input
                 type="text"
                 name="name"
                 id="name"
                 placeholder="Your Name"
               />
             </div>
-            <div>
+            <div class="pad-btm">
               <input
-                type="text"
+                type="number"
                 name="score"
                 id="score"
                 placeholder="Your Score"
               />
             </div>
             <div>
-              <input type="button" value="Submit" />
+              <input type="button" id="gameBtn" value="Submit" />
```

Comment:
## Status: ✔️ Project Approved ✔️ 
---
Excellent project, your code works as intended, I tested it and all the features work and are all perfect, I'm really proud to say you're approved and you don't have to worry, you can have a good weekend 😁😁✔
------
![giphy (1)](https://user-images.githubusercontent.com/54015740/151627648-6b4f3cfe-dab1-4160-8906-91a868957893.gif)